compiler/src/dmd/parse.d

```diff
@@ -1705,19 +1705,26 @@ class Parser(AST, Lexer = dmd.lexer.Lexer) : Lexer
      *      mixin Foo;
      *      mixin Foo!(args);
      *      mixin a.b.c!(args).Foo!(args);
-     *      mixin Foo!(args) identifier;
      *      mixin typeof(expr).identifier!(args);
```

Comment:
Only a small nit, I don't see a need to remove this comment at the top of the function.

Adding in the new syntax style would make it clear that this is also handled by parseMixin.

Reply:
Now done!